### ISSUE TITLE
Splitting PlantCLEF2022 dataset for train-val sets

### ISSUE THREAD

**Issue by duonglegiang** [xml94/PlantCLEF2022#4]
Thank you for great repository

Could you please explain how to divide the PlantCLEF2022 dataset into train and validation sets, and provide the list of files after the division?

Additionally, in make_val.py, why are files only copied to the validation set and not to the train and test sets?
```
for file in os.listdir(abs_dir):
        abs_src_file = osp.join(abs_dir, file)
        if num < total * ratio[0]:
            abs_tgt_file = osp.join(tgt_dir, 'val', dir, file)
            shutil.copyfile(abs_src_file, abs_tgt_file)
        elif num < total * (ratio[0] + ratio[1]):
            abs_tgt_file = osp.join(tgt_dir, 'train', dir, file)
        else:
            abs_tgt_file = osp.join(tgt_dir, 'test', dir, file)
        num += 1
```
